src/index.ts

```diff
@@ -170,7 +170,12 @@ export default function laravel(config?: string|string[]|Partial<PluginConfig>):
             return null
         },
         writeBundle() {
-            const manifestPath = path.resolve(resolvedConfig.root, resolvedConfig.build.outDir, 'manifest.json')
+            const manifestConfig = resolvedConfig.build.ssr ? resolvedConfig.build.ssrManifest : resolvedConfig.build.manifest;
+            if (manifestConfig === false) {
+                return;
+            }
+            const manifestFile = typeof manifestConfig === 'string' ? manifestConfig : 'manifest.json'
```

Comment:
Just wanna confirm that the default manifest for SSR is also `'manifest.json'`, as the docs don't specify.

Reply:
Looks like the default name in SSR is `ssr-manifest.json`

Reply:
Good catch - it should be `ssr-manifest.json`.

https://github.com/vitejs/vite/blob/0e20949dbf0ba38bdaefbf32a36764fe29858e20/packages/vite/src/node/ssr/ssrManifestPlugin.ts#L82